### PR TITLE
Issue #BIB-239: Logger service to send message to backend logs

### PR DESCRIPTION
**Pull request by cableman**

### CODE REVIEW

File: public/js/services/loggerService.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
+/**
+ * @file
+ * Expose logging messages to the backend logs.
+ *
+ * @TODO: If in debug mode also log to console.
```
Comment (tuj) [inline public/js/services/loggerService.js:5]: Yeah... also what I thought was missing

File: public/js/services/userService.js [hunk 2]
```diff
@@ -8,14 +8,17 @@
...
+angular.module('BibBox').service('userService', ['$q', '$timeout', '$location', 'config', 'proxyService', 'userTrackerService', 'loggerService',
+  function ($q, $timeout, $location, config, proxyService, userTrackerService, loggerService) {
     'use strict';
 
     this.username = null;
     this.password = null;
     this.loggedIn = false;
 
+
+    proxyService.emit('logger.frontend', 'TEST TEST');
```
Comment (tuj) [inline public/js/services/userService.js]: Remove :)

**Review by tuj — CHANGES_REQUESTED** [review]

File: public/js/services/loggerService.js (hunk 1 above)
Comment (cableman) [reply public/js/services/loggerService.js:5]: Yes, just not sure if we have a debug mode in the front-end?

File: public/js/services/userService.js (hunk 2 above)
Comment (cableman) [reply public/js/services/userService.js]: done

File: public/js/services/loggerService.js (hunk 1 above)
Comment (cableman) [reply public/js/services/loggerService.js:5]: Done

**Review by tuj — APPROVED** [review]

**Pull request merged**